Sources/SwiftDocC/Infrastructure/External Data/Deprecated/DocumentationContext+DeprecatedResolvers.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension DocumentationContext {
```

Comment:
I like how these are bundled up for easy removal, maybe we should have a more centralized spot for these?

Reply:
Maybe. I think it's fine to add new files for different groups of deprecated API as the need comes up.